proto/storage/cve.proto

```diff
@@ -29,6 +29,13 @@ enum Source {
   SOURCE_NVD = 3;
 }
 
+// ScoreVersion can be deprecated ROX-26066
+enum ScoreVersion {
+  V2 = 0; // No unset for automatic backwards compatibility
+  V3 = 1;
+  UNKNOWN_VERSION = 2;
```

Comment:
Was this enum added for 4.6 or did it exist prior to your other changes?  If it existed prior to 4.6, I would strongly suggest not changing the name.  if you added it in your previous 4.6 changes you can simply ignore and resolve this comment.

Reply:
@dashrews78  I have not changed the name. just moved it out of CVEInfo message so that I can reuse it in ImageCVE message

Reply:
Not true according to the diff, it was `UNKNOWN = 2;` before and now it is `UNKNOWN_VERSION = 2;`.  If `ScoreVersion` existed prior to 4.6 you should not change the value of that enum.  If `ScoreVersion` was added in 4.6 then changing the enum is fine.

Reply:
Why couldn't you pull `ScoreVersion` out and simply leave `UNKNOWN` as `UNKNOWN`?  Was that causing a conflict elsewhere?

Reply:
yes there are other scoreversion enum defined inside other message https://github.com/stackrox/stackrox/blob/master/proto/storage/cve.proto#L52